join/join-testnet-with-binaries.sh

```diff
@@ -53,8 +53,8 @@ addPeers() {
 }
 
 # override OS with amd64 for x86 arch
-if [ "x86_64" =  "$OS" ]; then
-  OS=amd64
+if [ "x86_64" =  "$ARCH" ]; then
```

Comment:
fix the comment

Reply:
comment is correct, we are overriding os with amd64 when its x86

Reply:
oh nvm